core/math/projection.cpp

```diff
@@ -915,7 +915,12 @@ Projection::operator String() const {
 	String str;
 	for (int i = 0; i < 4; i++) {
 		for (int j = 0; j < 4; j++) {
-			str += String((j > 0) ? ", " : "\n") + rtos(columns[i][j]);
+			if (j > 0) {
+				str += ", ";
+			} else if (i > 0) {
+				str += "\n";
+			}
+			str += rtos(columns[i][j]);
```

Comment:
Maybe that's material for a separate PR in the end, but I think we should change Projection's string representation to be similar to the other matrices.

So like copied from `Basis` but with `X`, `Y`, `Z`, `W`.

All other Variants print on a single line so this is the odd one out.

Reply:
So to be clear I'd suggest undoing the changes to add the Projection test here, and make another PR for the above.

Reply:
I've done this myself in #99563, so you can remove the Projection stuff from this PR.